lru_cache/lru_cache.py

```diff
@@ -30,4 +41,20 @@ def get(self, key):
     the newly-specified value.
     """
     def set(self, key, value):
```

Comment:
Straight forward implementation of LRU

Write some comments on your though process of solving this.